packages/renderer/src/lib/network/CreateNetwork.svelte

```diff
@@ -0,0 +1,321 @@
+  } finally {
+    createNetworkInProgress = false;
+  }
+}
+
+function parseKeyValueArray(list: string[]): Record<string, string> | undefined {
+  const validPairs: [string, string][] = [];
+  for (const entry of list.map(s => s.trim()).filter(Boolean)) {
+    const [key, value] = entry.split('=');
+    if (key && value) validPairs.push([key.trim(), value.trim()]);
+  }
+  return validPairs.length > 0 ? Object.fromEntries(validPairs) : undefined;
+}
+
+function end(): void {
+  // redirect to the networks page
+  router.goto('/networks');
+}
+
+let providerConnections = $derived(
+  $providerInfos.reduce<ProviderContainerConnectionInfo[]>((acc, provider) => {
+    const startedConnections = provider.containerConnections.filter(connection => connection.status === 'started');
+    return acc.concat(startedConnections);
+  }, []),
+);
+let selectedProvider = $derived(providerConnections.length > 0 ? providerConnections[0] : undefined);
+$effect(() => {
+  networkInfo.selectedProvider = selectedProvider;
+});
+
+// Auto-populate default subnet for ipvlan to help beginners
+$effect(() => {
+  // Only auto-populate when switching TO ipvlan from another driver
+  if (networkInfo.driver === 'ipvlan' && previousDriver !== 'ipvlan' && !networkInfo.subnet) {
+    // eslint-disable-next-line sonarjs/no-hardcoded-ip
+    networkInfo.subnet = '192.168.210.0/24';
+    // eslint-disable-next-line sonarjs/no-hardcoded-ip
+    networkInfo.gateway = '192.168.210.1';
+  }
+  previousDriver = networkInfo.driver;
+});
+
+let hasInvalidFields = $derived(!networkInfo.networkName || !networkInfo.selectedProvider);
+</script>
+
+<EngineFormPage
+  title="Create a Network"
+  inProgress={createNetworkInProgress}
+  showEmptyScreen={providerConnections.length === 0 && !createNetworkInProgress}>
+  {#snippet icon()}
+    <Icon icon={faNetworkWired} class="fa-2x" />
+  {/snippet}
+  {#snippet content()}
+    <div class="space-y-6">
+      <div hidden={createNetworkInProgress}>
+        <label for="networkName" class="block mb-2 font-semibold text-[var(--pd-content-card-header-text)]"
+          >Name <span class="text-red-500">*</span></label>
+        <Input
+          bind:value={networkInfo.networkName}
+          name="networkName"
+          id="networkName"
+          placeholder="Network name"
+          required
+          class="w-full" />
+      </div>
+
+      <div hidden={createNetworkInProgress}>
+        <label
+          for="labels"
+          class="block mb-2 font-semibold text-[var(--pd-content-card-header-text)]"
+          >Labels (key=value):</label
+        >
+      
+        {#each networkInfo.labels as _, index (index)}
+          <div class="flex flex-col w-full py-1">
+            <div class="flex flex-row justify-center items-center w-full">
+              <Input
+                bind:value={networkInfo.labels[index]}
+                placeholder="e.g., env=prod or app=backend"
+                class="ml-2 w-full" />
```

Comment:
```suggestion
                class="w-full" />
```

The margin left here is not required

<img width="229" height="265" alt="Image" src="https://github.com/user-attachments/assets/9dbdf4fa-e2a1-4600-bce4-aa00e2be5498" />